### ISSUE TITLE
Flip variables values

### ISSUE THREAD

**Issue by fabercancio** [elchininet/postcss-rtlcss#65]
Check [this issue](https://github.com/vkalinichev/postcss-rtl/issues/56) in [postcss-rtl repo](https://github.com/vkalinichev/postcss-rtl).

The value of the variables is not flipped, for example:

```css
:root {
  --padding: 1rem 1rem 1.5rem 1.5rem;
}

.my-class {
  padding: var(--padding);
}
```

Is compiled to:

```css
:root {
  --padding: 1rem 1rem 1.5rem 1.5rem;
}

.my-class {
  padding: var(--padding);
}
```

**Comment by elchininet** [comment]
Curious, a colleague requested exactly the same recently :)

In my opinion, this should be implemented in `RTLCSS`, so [I submitted a pull request](https://github.com/MohammadYounes/rtlcss/pull/223). I'll update this issue once that pull request gets accepted.

**Comment by fabercancio** [comment]
Glad to hear that!
Thanks for the update!

**Comment by elchininet** [comment]
Hi @fabercancio,

A new version of RTLCSS has released (3.2.0), so [I've added a new option](https://github.com/elchininet/postcss-rtlcss/pull/80) to cover your use case.

You can use version 3.3.4 of `postcss-rtlcss` which already supports this option.

Regards!

**Comment by fabercancio** [comment]
Thank you @elchininet!
